### PR TITLE
[CARBONDATA-656]Simplify the carbon session creation.

### PR DESCRIPTION
**Pull request by ravipesala**
Following is the way of creating carbon session in java code and spark-shell

1. Spark Shell
  Start spark shell
  `$./spark-shell --jars <carbon-jar-path>`
 
  Just execute following commands to create carbon session
  ```
   scala>import org.apache.spark.sql.SparkSession
   scala>import org.apache.spark.sql.CarbonSession._
   scala> val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession()
  ```
 After executing above commands CarbonSession is created as carbon

2.  With Java Code
Please refer examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/646/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/647/

**Comment by ravipesala** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/648/

File: examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala [hunk 1]
```diff
@@ -53,11 +43,8 @@ object CarbonSessionExample {
       .builder()
       .master("local")
       .appName("CarbonExample")
```
Comment (chenliang613) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala]: Please change the appName to "CarbonSessionExample"

File: integration/spark2/src/main/scala/org/apache/spark/sql/test/Spark2TestQueryExecutor.scala [hunk 2]
```diff
@@ -53,9 +52,7 @@ object Spark2TestQueryExecutor {
     .appName("CarbonExample")
```
Comment (chenliang613) [inline integration/spark2/src/main/scala/org/apache/spark/sql/test/Spark2TestQueryExecutor.scala]: Please change the appName.

**Comment by chenliang613** [comment]
please update SparkSessionExample also.

File: examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala (hunk 1 above)
Comment (ravipesala) [inline examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala]: Ok

File: integration/spark2/src/main/scala/org/apache/spark/sql/test/Spark2TestQueryExecutor.scala (hunk 2 above)
Comment (ravipesala) [inline integration/spark2/src/main/scala/org/apache/spark/sql/test/Spark2TestQueryExecutor.scala]: OK

**Comment by ravipesala** [comment]
retest this please

**Comment by chenliang613** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/663/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**